CMake/cdat_modules_extra/checkout_testdata.cmake

```diff
@@ -146,7 +146,7 @@ endif()
 
 # 5) Update the remote branches available on the testdata repo.
 execute_process(COMMAND
-  "${GIT_EXECUTABLE}" fetch --update-shallow --depth=1
+  "${GIT_EXECUTABLE}" fetch --depth=1  
```

Comment:
@aashish24 I think on his mac with old github it doesn't work.Similar to #1355 also see #1427